charts/pwa/docs/migrate-to-0.8.0.md

```diff
@@ -5,7 +5,50 @@
 The `ingress` and `ingresssplit` configurations were combined to only support one `ingress` object that can declare multiple instances.
 We also dropped support for supplying paths because it is not working for the PWA standard deployment.
 
-Old:
+> [!IMPORTANT]
+> When migrating from a previous PWA Helm Chart version to 0.8.0 it is important to remove the existing Ingress object from the cluster beforehand.
+> Because of the differently configured path configuration in the template it cannot be updated but needs to be recreated.
+> Please contact the OPS department for that.
+>
+> This is not relevant for new deployments, only for existing Ingress objects that are migrated.
+
+With this changed `ingress` declaration format a basic `ingress` configuration looks like the following example.
+
+```yaml
+ingress:
+  enabled: true
+  className: nginx
+  instances:
+    ingress:
+      annotations:
+        kubernetes.io/tls-acme: "false"
+      tlsSecretName: tls-star-pwa-intershop-de
+      hosts:
+        - host: pwa.example.local
```

Comment:
I think essential is only:

```yaml
ingress:
  enabled: true
  instances:
    ingress:
      hosts:
        - host: pwa.example.local
```

Reply:
Why leave the tls Secret away? The more detailed the guide is - the better :) And it should also be comprehensive from OPS Point of view, you always protect the hosts with a certificate there.